Answers/hw03_takport.py

```diff
@@ -0,0 +1,116 @@
+# Author: Tobore Takpor
+# Username: takport
+#
+# Assignment:  HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: To continue practicing creating and using functions. More practice on using the turtle library. Learn about how computers represent colors. Learn about source control and Git.
+
+# Google Doc Link: https://docs.google.com/document/d/1iM0_w1wRQGrA8l-tM-broe1hVOoxkpgx4ORivJB3Z8Y/edit?usp=sharing
+#
+#################################################################################
+
+
+import turtle
+
+
+def draw_flower():
+    """
+    Draws a flower with petals using an unnamed RGB color.
+    """
+    turtle.colormode(255)  # Set the color mode to RGB
```

Comment:
You're using the turtle library inappropriately. 

You should have passed a turtle into this function:

```
def draw_flower(t):
    ...
```

When you called the `draw_flower()` method in main(), you should have passed a turtle into it there: 

```
t = turtle.Turtle()
draw_flower(t)
```

Reply:
This issue exists for every function